docs/contributing/index.rst

```diff
@@ -3,59 +3,33 @@
 Contributing to Weblate
 =======================
 
+Language
+--------
 
-There are dozens of ways to contribute in Weblate. Any help is welcomed, be it
-coding, graphics design, documentation or sponsorship:
+`Translate <https://hosted.weblate.org/projects/weblate/>`_ Weblate itself,
+and point out errors in source strings while you do. Suggest source string changes, or
+click "Source string location" in the lower right of the regular editor to have
+a go yourself.
 
-* :doc:`issues`
-* :doc:`start`
-* :ref:`translating-weblate`
-* :ref:`contributing-docs`
-* :ref:`funding-weblate`
+Here in the documentation, click :guilabel:`Edit on GitHub` atop the page to do the same thing.
+`Open an issue https://github.com/WeblateOrg/weblate/issues/new?assignees=comradekingu&labels=ux&template=bug_report.md>`_
+if you know something is wrong or have an issue with the documentation, but not how to fix it.
+Remove the template stuff; a link and description will do.
 
-.. _translating-weblate:
+Graphics
+--------
+The visuals are in https://github.com/WeblateOrg/graphics
```

Comment:
This overlaps with https://docs.weblate.org/en/latest/contributing/about.html#project-logos

Reply:
This PR removes layers and layers of overlap. The real question is why pointing to the logos suggests they can be worked on, or details why. Listing that in "about" is usually done for people to use them (for which there are no stated terms). It is sort of a media-pack, only there is no write-up to use. (Though that fell out of fashion with SEO-ing, some basic history would do nicely. (Like linking to the Wikipedia article.) That is a good idea for writing _less_ documentation, in a more honest way.